docs/api/math/Vector2.html

```diff
@@ -175,6 +175,12 @@ <h3>[method:Float dot]( [param:Vector2 v] )</h3>
 		<p>
 		Calculates the [link:https://en.wikipedia.org/wiki/Dot_product dot product] of this
 	  vector and [page:Vector2 v].
+        </p>
```

Comment:
Please avoid mixing hard tabs with soft tabs. This will only mess up the formatting.